UserDetails/views_user_settings.py

```diff
@@ -29,13 +29,13 @@ def post(self, request, *args, **kwargs):
         })
 
         if context['user_form'].is_valid() and \
-            context['dining_form'].is_valid() and \
-            context['association_links_form'].is_valid():
+                context['dining_form'].is_valid() and \
```

Comment:
Why the increment?

Reply:
I'm just using the built-in PyCharm code formatter here. In this case it's obviously done to have a visual indication of the if statement (consisting of 3 lines) and the first statement after the if on the 4th line.